tests/mgbench/client.cpp

```diff
@@ -398,8 +400,12 @@ void ExecuteWorkload(
     final_duration += worker_duration[i];
   }
 
+  auto end_workload = std::chrono::steady_clock::now();
+  auto workload_duration = std::chrono::duration_cast<std::chrono::duration<double>>(end_workload - start_workload);
+
   final_duration /= FLAGS_num_workers;
   nlohmann::json summary = nlohmann::json::object();
+  summary["workload_duration"] = workload_duration.count();
```

Comment:
@Josipmrden Not sure did you plan to use these measurements anywhere. I know you used them for debugging, but "workload duration" is a bit wrong since it includes thread init +  connection time.  And we have workload as an important reference so I would call this something like **total_duration**, later we can have we have "thread_init_duration" "connection_duration" + "workload_duration" . If you are not going to use it, feel free to kick it out.

Reply:
will rename, but it can come in handy since we don't know how much was the total time of execution, so will rename it